community/rust/APKBUILD

```diff
@@ -16,7 +16,7 @@ license="Apache-2.0 BSD ISC MIT"
 # gcc is needed at runtime just for linking. Someday rustc might invoke
 # the linker directly, and then we'll only need binutils.
 # See: https://github.com/rust-lang/rust/issues/11937
-depends="$pkgname-stdlib=$pkgver-r$pkgrel gcc llvm-libunwind-dev musl-dev"
```

Comment:
How much testing did you do after making this change? I was under the impression that on musl, rustc requires llvm-libunwind-dev to be present during compilation (but maybe I'm wrong, of course, or maybe it's a makedepends and not a normal dependency)